### PR TITLE
Display Post Image and Add Default Archtypes

### PR DESCRIPTION
**Pull request by okabrionz**
Post image not shown, and I update README.md too
image

### CODE REVIEW

**Comment by dim0627** [comment]
awesome!

> Post image not shown

Really?
I could not reproduced it.
But your implementation is great :)

Please remove extra codes.

* https://github.com/dim0627/hugo_theme_aglaus/blob/master/layouts/_default/baseof.html#L19
* https://github.com/dim0627/hugo_theme_aglaus/blob/master/layouts/partials/styles.css#L275

And default thumbnail might does not working 🤔 

Extra borders shown 😭 
Why changed`layouts/_default/summary.html`?

screen shot 2017-09-13 at 9 06 21

**Review by okabrionz — COMMENTED** [review]
All done, Let's do this 😅

**Comment by okabrionz** [comment]
I suggest changing Theme name to Aglaus AMP (without Hugo).
Why without Hugo? because this theme is only can be running with Hugo right?.

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html ⚡>
+<html ⚡ lang="lang">
```
Comment (dim0627) [inline layouts/_default/baseof.html:2]: why it is needed?

File: layouts/partials/share.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
-<aside class="p-share">
-  <a href="http://b.hatena.ne.jp/add?mode=confirm&url={{ .Permalink }}&title={{ .Title }}" title="はてなブックマーク" class="ht" target="_blank" rel="nofollow"></a>
-  <a href="http://www.facebook.com/sharer.php?u={{ .Permalink }}&t={{ .Title }}" title="Facebookでシェア" class="fb" target="_blank" rel="nofollow"></a>
-  <a href="http://twitter.com/intent/tweet?url={{ .Permalink }}&text={{ .Title }}&tw_p=tweetbutton" title="Twitterでシェア" class="tw" target="_blank" rel="nofollow"></a>
-  <a href="https://plus.google.com/share?url={{ .Permalink }}" title="Google Plusでシェア" class="gp" target="_blank" rel="nofollow"></a>
-  <a href="http://line.me/R/msg/text/?{{ .Title }} {{ .Permalink }}" title="LINEでシェア" class="ln" target="_blank" rel="nofollow"></a>
-</aside>
+<amp-social-share width="30" height="30" data-param-text="{{ .Title }}" data-param-attribution="Hugo AMP Aglaus" type="email"></amp-social-share>
+<amp-social-share width="30" height="30" data-param-text="{{ .Title }}" data-param-attribution="Hugo AMP Aglaus" type="facebook" data-param-app_id="254325784911610"></amp-social-share>
```
Comment (dim0627) [inline layouts/partials/share.html:2]: what is `254325784911610`?

File: layouts/_default/summary.html [hunk 3]
```diff
@@ -1,53 +1,49 @@
 <article class="single article-{{ .Source.Path | md5 }}">
   <div class="header-wrapper">
-    <a href="{{ .Permalink }}" class="thumbnail" title="{{ .Title }}"></a>
+    <section>
+      {{ $params := .Params }} {{ range $key, $value := $.Site.Taxonomies }} {{ if isset $params $key }}
+      <ul class="p-facts">
```
Comment (dim0627) [inline layouts/_default/summary.html:5]: Why move this section?

**Comment by dim0627** [comment]
I added some comments.
But I'm sorry, I think this PR should not merge and you can fork new theme.
Because this changes beyond "Display Post Image and Add Default Archtypes" purpose.
And I guess this changes almost solve using by Hugo override system.
[Hugo \| Customize a Theme](https://gohugo.io/themes/customizing/#override-template-files)

Regards

**Pull request closed without merging**